dataless-classifier/src/main/java/edu/illinois/cs/cogcomp/datalessclassification/util/Utilities.java

```diff
@@ -0,0 +1,40 @@
+package edu.illinois.cs.cogcomp.datalessclassification.util;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Utilities {
```

Comment:
Rename this something that indicates it's specific to the dataless package? Say `DatalessUtilities`?